docs/Dockerfile

```diff
@@ -5,7 +5,7 @@ MAINTAINER Mu Li <muli@cs.cmu.edu>
 # First, build MXNet binaries (ref mxnet/docker/cpu/Dockerfile)
 #
 
-RUN apt-get update && apt-get install -y build-essential git libopenblas-dev libopencv-dev
+RUN apt-get update && apt-get install -y build-essential git libopenblas-dev liblapack-dev libopencv-dev
```

Comment:
Please note that for Ubuntu 14.04 LTS, the `libopenblas.so` within `libopenblas-base` or `libopenblas-dev` only packed in the BLAS routines but not LAPACK. Only from Ubuntu 16.04 LTS, the `libopenblas.so` in the official packages contain all the BLAS and LAPACK routines.

If we install `liblapack-dev` that's the unoptimised Netlib version which bear no relation with OpenBLAS.

To rectify this, we have to base the test on the latest Ubuntu LTS (aka 16.04) Image, remove `liblapack-dev` -- using just `libopenblas-base` or `libopenblas-dev` would suffice to provide all the BLAS/LAPACK routines.

(If we install `libblas-dev` or `liblapack-dev` that's the unoptimised Netlib version. We could do `sudo update-alternatives --display liblapack.so` to check.)